code/modules/clothing/suits/jobs.dm

```diff
@@ -72,6 +71,12 @@
 		"Vox" = 'icons/mob/species/vox/suit.dmi'
 		)
 
+/obj/item/clothing/suit/mantle/armor/captain
+	name = "captain's cloak"
+	desc = "An armor-plated piece of fashion for the ruling elite. Protect your upper half in style."
+	icon_state = "capmantle"
+	item_state = "capmantle"

```

Comment:
armor should be `armor = list(melee = 50, bullet = 40, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)`